spec/models/dialog_group_spec.rb

```diff
@@ -21,4 +21,45 @@
       expect(dialog_group.dialog_fields).to be_empty
     end
   end
+
+  context '#update_dialog_tabs' do
+    before(:each) do
```

Comment:
Very minor, but you don't need the `(:each)` here, it's implied if you just use `before do`